src/Toml/Bi/Map.hs

```diff
@@ -52,6 +52,8 @@ module Toml.Bi.Map
        , _Set
        , _IntSet
        , _HashSet
+       , _KeyText
+       , _KeyString
```

Comment:
Let's add a separate section in the documentation for `Key` mappings. We will probably have more of them in the future.
```suggestion

         -- * Bidirectional mappings for 'Key'
       , _KeyText
       , _KeyString
```